### PR TITLE
v5.0.x: docs: add some info about HPE CXI ofi provider

### PR DESCRIPTION
**Pull request by hppritcha**
and also make a statement about the OFI BTL more accurate.

Related to #12038

Signed-off-by: Howard Pritchard <howardp@lanl.gov>
(cherry picked from commit 2718732fc18af7ce00ebd62c002a85282766b38b)

### CODE REVIEW

**Review by jsquyres — APPROVED** [review]

File: docs/tuning-apps/networking/ofi.rst [hunk 1]
```diff
@@ -76,6 +78,11 @@ list. For example:
 
    shell$ ompi_info --param mtl ofi --level 9
 
+.. important:: When using the HPE CXI provider and ``mpirun`` as the job launcher,
```
Comment (rhc54) [inline docs/tuning-apps/networking/ofi.rst:81]: Just to be clear: this isn't entirely accurate. You could also simply add `--map-by :nouselocal` to the `mpirun` cmd line.

Comment (hppritcha) [reply docs/tuning-apps/networking/ofi.rst:81]: yeah, if you want to get charged for N+1 vs N nodes where N is the number of nodes available for your MPI processes.

Comment (rhc54) [reply docs/tuning-apps/networking/ofi.rst:81]: I understand your point - however, your text doesn't indicate that you have a choice. You state that you *must* use the specified option - and that simply isn't true.

Comment (jsquyres) [reply docs/tuning-apps/networking/ofi.rst:81]: That's a fair point.  It's probably worth pointing out the subtlety here -- i.e., what does this parameter mean and do, vs. what the user's options are if they don't use this MCA param.

**Review by jsquyres — APPROVED** [review]

**Pull request merged**